app/src/forms/attestations/emailService.js

```diff
@@ -0,0 +1,251 @@
+        return await chesService.merge(data);
+      } else {
+        return false;
+      }
+    } catch (err) {
+      log.error('sendSubmissionEmail', `Error: ${err.message}.`);
+      log.error(err);
+      throw err;
+    }
+  }
+
+  async _getStatusAssignmentEmailContexts(settings, statusUpdate) {
+    const submission = await this._dataService.readSubmission(statusUpdate.submissionId);
+    const statusCodes = await this._dataService.readCurrentStatusCodes();
+    const statusCode = statusCodes.find(x => x.code === statusUpdate.code);
+    return [
+      {
+        context: {
+          confirmationNumber: submission.confirmationId,
+          business: submission.business.name,
+          status: statusCode.display,
+          message: settings.config.message,
+          title: settings.config.title,
+          messageLinkText: settings.config.messageLinkText,
+          messageLinkUrl: `${settings.config.messageLinkUrl}/${submission.submissionId}`
+        },
+        to: [statusUpdate.assignedToEmail]
+      }
+    ];
+  }
+
+  async sendStatusAssignmentEmail(statusUpdate) {
+    try {
+      const settings = await this._dataService.readSettings('statusAssignmentEmail');
+      if (settings && settings.enabled && statusUpdate.assignedToEmail) {
+        if (!this._statusAssignmentEmailBody) {
+          this._statusAssignmentEmailBody = fs.readFileSync(`${this._assetsPath}/${settings.config.template}`, 'utf8');
+        }
+
+        const contexts = await this._getStatusAssignmentEmailContexts(settings, statusUpdate);
+
+        const data = {
+          body: this._statusAssignmentEmailBody,
+          bodyType: 'html',
+          contexts: contexts,
+          ...settings.config
+        };
+        return await chesService.merge(data);
+      } else {
+        return false;
+      }
+    } catch (err) {
+      log.error('sendStatusAssignmentEmail', `Error: ${err.message}`);
+      log.error(err);
+      throw err;
+    }
+  }
+
+  async _getAccessRequestedEmailContexts(settings, accessRequest, to) {
+    return [
+      {
+        context: {
+          title: settings.config.title,
+          userInfo: `${accessRequest.firstName} ${accessRequest.lastName} - ${accessRequest.email}`,
+          message: settings.config.message,
+          messageLinkText: settings.config.messageLinkText,
+          messageLinkUrl: `${settings.config.messageLinkUrl}`
+        },
+        to: to
+      }
+    ];
+  }
+
+  async sendAccessRequestedEmail(accessRequest) {
+    try {
+      const settings = await this._dataService.readSettings('accessRequestedEmail');
+      if (settings && settings.enabled) {
+        if (!this._accessRequestedEmailBody) {
+          this._accessRequestedEmailBody = fs.readFileSync(`${this._assetsPath}/${settings.config.template}`, 'utf8');
+        }
+        const to = settings.config.to.split(',').filter(x => x);
+
+        const contexts = await this._getAccessRequestedEmailContexts(settings, accessRequest, to);
```

Comment:
settings is already passed to the _getBlahBlah fxns. So the 'to' formatting could be done down in those instead of passing in as well right?

Reply:
next PR.